server/src/test/java/org/apache/uniffle/server/buffer/ShuffleBufferManagerTest.java

```diff
@@ -621,5 +621,10 @@ private void waitForFlush(ShuffleFlushManager shuffleFlushManager,
         fail("Flush data time out");
       }
     } while (committedCount < expectedBlockNum);
+
+    // Need to wait for `event.doCleanup` to be executed
+    // to ensure the correctness of subsequent checks of
+    // `shuffleBufferManager.getUsedMemory()` and `shuffleBufferManager.getInFlushSize()`.
```

Comment:
We use `awaitility` here better.
await().atMost(5, SECONDS).until(shuffleBufferManager.getUsedMemory() == 0);

Reply:
SGTM. 👍